docs/api/deriva.seo.rst

```diff
@@ -0,0 +1,15 @@
+deriva.seo package
+=======================
```

Comment:
I know it sounds like nit-picking but `spinx-autodoc` will actually emit a warning when the `=` line string is too long/short for the title. Lame, I know. Not a biggie for now.